### DISCUSSION TITLE
4.0.2

### DISCUSSION THREAD

**Discussion by nalexandru** [nalexandru/BlenderCompat#72] [Announcements]
Blender 4.0.2, commit [9be62e85b727](https://projects.blender.org/blender/blender/commit/9be62e85b727).
Build environment:

    Visual Studio 2019, version 16.11.32
    Windows SDK 10.0.22621
    NVIDIA CUDA 11.8
    NVIDIA OptiX 7.3

AMD HIP and SM3.0 kernel binaries copied from the official release.

The required OpenGL version is now 4.3 and Intel HD4000 support was dropped.
Please read the Graphics Cards section of the release notes: [https://wiki.blender.org/wiki/Reference/Release_Notes/4.0](https://wiki.blender.org/wiki/Reference/Release_Notes/4.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/nalexandru/BlenderCompat/releases/tag/4.0.2'>4.0.2</a>.</em>

**Comment by atekdigital** [comment]
Whats going wrong plz let me know. I was using 3.5 without any issue or error in windows 7. And after downloading 4.0.1 version it open without any issue or error, when i hit load 3.5 setting button, blender took time and shutdown, When i try to open new version it did not, with the old version also blender unable to start (launch). Whats going wrong i try to delete all blender foundation folder and registry still unable to open new as well my 3.5 working version. How to find why blender not start, as there is no error or message. 
i tried to blender_debug_gpu open in cmd press any key a empty notepad came. with this file also i did same result
blender_debug_gpu_glitchworkaround
blender_debug_log
blender_factory_startup

Plz help from 2 weeks i did not use blender. Thx

**Reply by nalexandru** [reply]
Please create an issue and upload the log files from both 3.5 and 4.0.

**Reply by atekdigital** [reply]
How can u plz explain the steps
